src/components/mdSidenav/mdSidenav.vue

```diff
@@ -45,7 +56,71 @@
         } else {
           this.open();
         }
+      },
+      handleTouchStart(event) {
+        const positionX = event.touches[0].clientX - this.mountedRect.left;
+        const positionY = event.touches[0].clientY - this.mountedRect.top;
+
+        if (positionX < 0 || positionX > this.mountedRect.left + this.mountedRect.width) {
```

Comment:
Maybe will be good to create three methods with the conditions inside those if statements, and then merge then to only one if, like that:
 
``` js
if (this.methodOne() || this.methodTwo() || this.methodThree()) {
  return;
}
```